src/ui/wxWidgets/wxUtilities.cpp

```diff
@@ -364,6 +364,41 @@ void wxUtilities::NotifyIfUnsupported(enum Feature feature, wxWindow* window)
 #endif
 }
 
+/**
+ * Checks if the environment variable PWS_ADVANCED_SCROLL
+ * is set to a value greater than zero.
+ * 
+ * Using this variable, the user can activate the option to
+ * position the search result higher up in the tree view and,
+ * in the case of an individual configuration of the scrollbar
+ * size, to prevent it from being obscured by the scrollbars.
+ */
+bool wxUtilities::IsAdvancedScrollEnabled()
```

Comment:
Why use an env variable rather than a UI config option or even make this behavior apply always?